src/file/file.controller.ts

```diff
@@ -24,7 +26,7 @@ export class FileController {
   @UseGuards(AuthGuard('jwt'))
   @ApiOperation({ summary: 'Create a new file' })
   @ApiCreatedResponse({ type: FileResponse })
-  @ApiUnauthorizedResponse({ description: 'Not logged in' })
+  @ApiUnauthorizedResponse({ description: 'Not logged in or unverified' })
```

Comment:
I would change the error message to simply: 
```suggestion
  @ApiUnauthorizedResponse({ description: 'User not authorized' })
```